<launch>
  <arg name="hand_model" default="AG-2E"/>

  <!-- via USB -->
  <!--<param name="Connect_port" type="string" value="/dev/DH_hand"/>-->

  <!-- via TCP -->
  <node pkg="dh_hand_driver" type="hand_controller" name="hand_controller" output="screen">
    <param name="hand_model" type="string" value="$(arg hand_model)"/>
    <param name="connect_port" type="string" value="192.168.1.30:8888"/>
    <param name="data_timeout" type="double" value="0.05"/>
  </node>

</launch>
